### DISCUSSION TITLE
Give ability to server administartor  to undress all players who no longer visit the server.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#12122] [Ideas and suggestions]
The ability to undress all players who no longer visit the server. Especially those who were banned.
It's very disappointing when a person enters one game, takes his equipment and then leaves the server forever.
This can be implemented as a console command that returns to the submarine all the inventory of a player with a specific name, IP address or steam ID

**Comment by Hazmat616** [comment]
Could also make a server setting that drops all items into a duffel bag when you leave. I’ve seen other games do similar.
